teams/vsp/teams/tools/frontend/cypress-migration-guide.md

```diff
@@ -0,0 +1,243 @@
+
+In Cypress, because everything in a test is executed inside of the browser, [fixtures](https://docs.cypress.io/api/commands/fixture.html#Syntax) are used to get access to data in tests.
+
+Fixtures should be stored in the app specific folder inside of `src/platform/testing/e2e-cypress/fixtures`. If the app specific folder does not exist feel free to create it.
+
+For example: `src/platform/testing/e2e-cypress/fixtures/hca` contains all the fixtures for the `hca` application. You can copy all the schema files used in current tests to the proper fixtures folder.
+
+You can inject the fixture into your test by doing:
+```javascript
+cy.fixture('hca/maximal-test.json').as('testData');
+```
+Cypress already knows to look in the `fixtures` folder, so you can use the application specific directory for the path to the fixture.
+
+To get access to the fixture, you can use `.get()`:
+
+```javascript
+cy.get('@testData').then(testData => {
+  cy.get('#root_firstName').type(testData.veteranFullName.first);
+});
+```
+
+### Mocking Data <a name="mocking-data"></a>
+
+Cypress allows you to [stub responses](https://docs.cypress.io/guides/guides/network-requests.html#Stubbing), which can avoid the need to use the mock api helpers.
+
+```javascript
+// Start a server to begin routing responses
+cy.server();
+
+// Stub request to get enrollment status
+cy.route({
+  method: 'GET',
+  url: '/v0/health_care_applications/enrollment_status*',
+  status: 404,
+  response: {
+    errors: [
+      {
+        title: 'Record not found',
+        detail: 'The record identified by  could not be found',
+        code: '404',
+        status: '404',
+      },
+    ],
+  },
+}).as('getApplication');
+
+// Stub request to submit application
+cy.route('POST', '/v0/health_care_applications', {
+  formSubmissionId: '123fake-submission-id-567',
+  timestamp: '2016-05-16',
+}).as('submitApplication');
+
+// Workaround to intercept requests made with fetch API.
+cy.on('window:before:load', window => {
+  delete window.fetch;
+});
+```
+Based on the above stubs, whenever the browser makes a `GET` request to `/v0/health_care_applications/enrollment_status*` or a `POST` request to `/v0/health_care_applications`, cypress will automatically stub the given response.
+
+All of your mock api calls using `mockData()` can be replaced with `cy.route()`. Note, be sure to always start the server with `cy.server()` before stubbing requests.
+
+### File Uploads <a name="file-uploads"></a>
+File uploads can be done using the [cypress file upload](https://github.com/abramenal/cypress-file-upload/tree/v3.5.3) plugin. For instructions on usage please visit [here](https://github.com/abramenal/cypress-file-upload/tree/v3.5.3#usage).
+
+**Note:** Version 4 is unstable to please follow instructions for v3.5.3.
+
+### Accessibility <a name="accessibility"></a>
+Just like most e2e tools, cypress has its own [axe-core plugin](https://github.com/avanslaars/cypress-axe) to test accessibility.
```

Comment:
It should deliver the same coverage as nightwatch because they are both using `axe` at their core. It may be helpful for us to go over the global `axe-core` settings in cypress with Trevor to see if he has any recommendations.